### DISCUSSION TITLE
Anyone can delete a detection?

### DISCUSSION THREAD

**Discussion by Sp4rkR4t** [mcguirepr89/BirdNET-Pi#648] [Q&A] [answered]
Daughter pointed out today that she could delete a detection I sent her without any login details, is this expected behavoir and is there a way to change it so you get the login prompt if you click the delete button on a detection?

**Comment by ehpersonal38** [comment, marked as answer]
That's definitely not expected behavior. If you're on the latest BirdNET-Pi update, then really the only cause I can think of is that she didn't get the login prompt because her browser already was logged in earlier (if her browser can also access "Tools" without being prompted for a login, then this confirms that she's indeed logged in).

**Reply by Sp4rkR4t** [reply]
That's the issue, she doesn't have any login details and I've never used her device to login, I can't seem to replicate this now I'm home on my PC though so will have to borrow her device to see if I can do this again.

**Reply by Sp4rkR4t** [reply]
Never mind, she got confused.
